src/Symfony/Component/Form/Tests/Extension/Validator/EventListener/ValidationListenerTest.php

```diff
@@ -66,7 +66,7 @@ protected function setUp()
 
     private function getConstraintViolation($code = null)
     {
-        return new ConstraintViolation($this->message, $this->messageTemplate, $this->params, null, 'prop.path', null, null, $code, new Form());
+        return $this->getMock('Symfony\Component\Validator\ConstraintViolation', array('getConstraint'), array($this->message, $this->messageTemplate, $this->params, null, 'prop.path', null, null, $code));
```

Comment:
no need to use a mock IMO. Just add an argument in the getter to decide whether the last argument should be the constraint or ``null``